### ISSUE TITLE
After palservice0.2.2.0 is updated, if us4ss is loaded and started, it cannot be started.

### ISSUE THREAD

**Issue by sunyink** [UE4SS-RE/RE-UE4SS#470]
**Branch or Release**
The Tag for the Release you are using, or the commit you used if you built from source yourself.
**[v3.0.1](https://github.com/UE4SS-RE/RE-UE4SS/releases/tag/v3.0.1) is.** ，https://github.com/UE4SS-RE/RE-UE4SS/commit/d935b5b23bac03b65c14ae38382b02007204cc2e

**Game and Engine Version**
Game where the bug occurred, and the UE engine version of the game.
**SteamCmd,Palservice v.0.2.2.0**

**Describe the bug**

**Originally, I had successfully set up a new server and loaded us4ss normally. But later that day, steam released the 0.2.2.0 update. After the server was upgraded, when starting the server's command line, it would not automatically display "Setting breakpad minidump AppID = 1623730". In this case, no client could connect. The server and client reported a timeout. I judged that the server was not started at this time. That is, when an error occurs, "Setting breakpad minidump AppID = 1623730" will not be displayed; but "REST API started on port 38212" will be displayed as usual. I deleted the "UE4SS.dll" file and everything on the server was normal, but I also sadly lost the Mod.**

**Mods directory**
A zip of your Mods folder if you've installed any mods that don't come with UE4SS by default

**CheatManagerEnablerMod : 1
ActorDumperMod : 0
ConsoleCommandsMod : 1
ConsoleEnablerMod : 1
SplitScreenMod : 0
LineTraceMod : 0
BPModLoaderMod : 1
BPML_GenericFunctions : 1
jsbLuaProfilerMod : 0**

No changes were made

**To Reproduce**
1. Launch game
2. Attempt UHT Dump
3. See error in log

**Expected behavior**

**Screenshots, UE4SS Log, and .dmp file**
If applicable, add screenshots to help explain your problem, and upload the log and .dmp files located in the Win64 directory.

**[logs - copy.zip](https://github.com/UE4SS-RE/RE-UE4SS/files/14994553/logs.-.copy.zip)
No .dmp file can be found in any subfolder under [\PalServer]**

**Desktop (please complete the following information):**
 - **OS: Win10 LSTC**

**Additional context**

**Comment by UE4SS** [comment]
This is probably the same as #452.
Please try this version: https://github.com/UE4SS-RE/RE-UE4SS/releases/download/experimental/UE4SS_v3.0.1-54-g258eeb1.zip

**Comment by Greelan** [comment]
Any indication when the fix in experimental will make it into a release, please?

**Comment by UE4SS** [comment]
> Any indication when the fix in experimental will make it into a release, please?

The only thing I'm aware of is this: https://github.com/Re-UE4SS/UEPseudo/pull/86#issuecomment-2043252599

TLDR: Not any concrete plans yet, but sooner rather than later is preferred even if it means some features not making it.

It's important to know that this is a hobby project, and that everyone that works on it does so on their own time prerogative, and that includes all of the maintainers.
A release will happen eventually, it's just a matter of people finding the time to do everything necessary for a proper release.

Here's the full text from the link above for everyone that doesn't have access, posted Apr 8:

###### Next release will definitely be a minor bump at least (3.1.0) given all cpp mods will need switching over to xmake and a couple changes that are slightly breaking (but I wouldn't be against a major version bump either).

###### Imo the next release should at least have datatable support and Linux port done (unless yang jumps out of the window), and for DT support there needs to be whatever dynamic type library integration as well. Plus cc is working on TSet/more TMap stuff so would be good to have those in that release too... but if it goes on for too long I would rather push for release now, more features later as I don't want a repeat of the 3.0.0 delays fiasco.

###### In regards to xmake change, at least a couple weeks before a planned deadline for next release, and announcement should be done to all c++ mod authors in the discord that tell them they should be aware of the change and can preemptively update their mods ready for the next release - which would also help iron out any remaining bugs or docs info for it.

**Comment by sunyink** [comment]
After using UE4SS_v3.0.1-54-g258eeb1.zip, the server can start normally. I continued to test for a while. In the past 24 hours, the server has not reported an error. Thank you.

In addition, I also installed this version on the client and found that creating a new stand-alone archive will cause a crash. I am still investigating this.

**Comment by UE4SS** [comment]
What do you mean by "creating a new stand-alone archive" ?

**Comment by sunyink** [comment]
> What do you mean by "creating a new stand-alone archive" ?

Hello, what I wanted to express before was“Start Game”→“Create New World”，like create a offline game.

**Comment by UE4SS** [comment]
Can you try this version ? https://github.com/UE4SS-RE/RE-UE4SS/issues/473#issuecomment-2063624357

**Comment by sunyink** [comment]
Great, the same mod configuration uses zDEV-UE4SS_v3.0.1-54-g3976965.zip, there is no crash, and both aspects of loading and new games can work.

Moreover, I investigated the cause of the previous crash. Today, the client's UE4SS_v3.0.1-54-g258eeb1 crashed when it coexisted with the "Teh Extra Base Area" Mod. After removing this only external Mod, the new game will not crash.

Teh Extra Base Area error report details: If you are interested in knowing: there was no crash when the client first installed the mod, but the crash occurred about 1 day later. Because the main body of the mod runs normally on the server and takes effect normally, the client mod only has the function of modifying the visual component radius of the blue line of the stronghold, so deleting the client mod has little impact on the gameplay. But I still found the corresponding dmp and log, if it can help.
[Uploading UECC-Windows-3D27E82749A1FE44B2CFB0BB022EE9FC_0000.zip…]()

**Comment by UE4SS** [comment]
> Great, the same mod configuration uses zDEV-UE4SS_v3.0.1-54-g3976965.zip, there is no crash, and both aspects of loading and new games can work.
> 
> Moreover, I investigated the cause of the previous crash. Today, the client's UE4SS_v3.0.1-54-g258eeb1 crashed when it coexisted with the "Teh Extra Base Area" Mod. After removing this only external Mod, the new game will not crash.
> 
> Teh Extra Base Area error report details: If you are interested in knowing: there was no crash when the client first installed the mod, but the crash occurred about 1 day later. Because the main body of the mod runs normally on the server and takes effect normally, the client mod only has the function of modifying the visual component radius of the blue line of the stronghold, so deleting the client mod has little impact on the gameplay. But I still found the corresponding dmp and log, if it can help. Uploading UECC-Windows-3D27E82749A1FE44B2CFB0BB022EE9FC_0000.zip…

Unfortunately that particular kind of dmp file is not useful to me.
The original issue has been solved so I'm closing this.
You should report this new problem to the mod author, they can make a new issue if they think it's a problem with UE4SS.
